src/av2/datasets/tbv/scene_clusters.py

```diff
@@ -0,0 +1,356 @@
+    47: ["e2Vvd8qpD57RA2ewZ40Sl1lwzvALFXfu__Summer_2020"],
+    48: ["zQ06g8U26AU3pRHhzNxLPm340nOybscg__Spring_2020"],
+    49: ["WY0cVNmhg7LtAs5Eny78Csltv2tbjdsd__Winter_2021"],
+    50: ["01bb304d7bd835f8bbef7086b688e35e__Summer_2019"],
+}
+
+DTW_CLUSTERS = {
+    1: ["XMbQ7jb2aVss95K5Ie9wTHK3LUTX8nz0__Spring_2020"],
+    2: ["9U3zmqrZK6BVF6yUmseXSe6P4U1h6jUZ__Summer_2020"],
+    3: ["ps87cXMi82mKmZ66R5iUVqLOcE9ZdYps__Summer_2020", "ouQqWBO1kAyPrbkIx7LcBVQgInxdntTq__Summer_2020"],
+    4: [
+        "lBEMpTsvTu1k0SjZYLeZdVVcxgyZO6Gz__Summer_2020",
+        "gsYZ0RMDMFZdRo1vmdW2Hp0Pn6hnKoIw__Autumn_2020",
+        "hAsGqwlkaBv1zyxXl2tE7uqg4epa36ir__Winter_2021",
+    ],
+    5: [
+        "5irmKvMAXQjJjUCC4VFT1FC7qsI8a4Kg__Summer_2020",
+        "vyWiJxFSWgVvk5Yd3KVeyGKauOLypJnC__Summer_2020",
+        "HGDGffqH58I2WAoFYKuTr1OYts3JWk4j__Summer_2020",
+        "PRRmJxg1eSDS95RprHX0M1Ord3OCMcsg__Summer_2020",
+        "s9aRUw1JDzPZPRmcx2h0wQCO2cerF66I__Autumn_2020",
+        "b86c6b2e0ed64f4f89f43c1b00391731__Autumn_2020",
+        "gu2UwLc2nU5YgX6tzcmfbqpPXWTngAOt__Autumn_2020",
+        "42d1b09f89cd4e1d866eff93fd05b80f__Autumn_2020",
+        "DKJk1QVisLulVf1jAvxijRTN33EBkgUE__Autumn_2020",
+        "Dgs3UYZvrcojLDw0QLWmD24dPn0jAXhC__Autumn_2020",
+    ],
+    6: ["aEEUxFn6Gm6XeqV4rLA9esoNpYUhs2Ql__Summer_2020"],
+    7: ["8pOtV4RsIeb0byPPCM6212tiiGNUvCyB__Summer_2020"],
+    8: [
+        "zRVrsPX6zQD7NKHIuBs4h5SgijdoNpKw__Autumn_2020",
+        "3V9lgpdSQfTMXj3HjG2yt2gDbkcTK0v1__Winter_2021",
+        "WwiugaY1H3aulOMXUCHbFTZKLDoqWuNP__Winter_2021",
+        "cAstq5i3iTdHouG6HZh90LdA6BMbTW2A__Winter_2021",
+    ],
+    9: ["GJ2Kd0JBW2QQKEst8XefgaKKKOjb5RoX__Summer_2020", "5c2322b6817b4bdd8a23a43244e712c0__Summer_2020"],
+    10: ["FJVcWS3nQWkh9XU4TJzmCvaOXlByM947__Summer_2020", "XE3HUDWFcKhT1LcGp8GS8HacmMmok9dc__Autumn_2020"],
+    11: ["MVnyJgZdDbxqbQu9h9Q8kSjbziatTgxa__Autumn_2020"],
+    12: ["rdWmCWPFNUuwwh9MBpEKzOEqJsmMuuyk__Winter_2021"],
+    13: [
+        "i47t2l9q6nJQNDFbD6iI0MrQ11yq4JNB__Autumn_2020",
+        "2135a974fc804227843bd33514afbf5a__Autumn_2020",
+    ],
+    14: ["G6HCUIYJ0crBy1BbY5z5zx1OtCA8NZF0__Summer_2020"],
+    15: ["75e8adad50a6324587265e612db3d165__Autumn_2019"],
+    16: [
+        "B1Ja9WqVlz5CAkSv2xsaiJK6KQq6egiT__Spring_2020",
+        "79O52mmzE8VvjKQy2P1U9qywm2tneeXS__Autumn_2020",
+        "sYzy2sKcm1qvahStIJf8h4Pl7HBEma7x__Autumn_2020",
+        "r86GRA2V3jgD7pswTLq6NLJJBQqq2EMs__Autumn_2020",
+    ],
+}
+
+PAO_CLUSTERS = {
+    1: ["KH70lsYpeBe1I2TqKoD4V5AKJXfcCfpO__Summer_2020"],
+    2: ["X3zP8oWJB4ivWYkbv6X9dw2mc1lRsceY__Summer_2020"],
+    3: ["TOQaS7eCZB2ns552jjhmQoPs2MbdkgC6__Summer_2020"],
+    4: [
+        "Nr6t0auYyTEC42fJNIqhkaSasyGjfV6E__Autumn_2020",
+        "jDONAwGVoVt9Ml4ZQW3FaTgDuhIlnAnt__Autumn_2020",
+        "cJiXgBbopBsjrK8AE9GuHcuJCAGsHodO__Autumn_2020",
+    ],
+    5: ["453e5558636338e3bf9b42b5ba0a6f1d__Summer_2020"],
+}
+
+TBV_LOG_CLUSTERS = {
```

Comment:
Could this be an `Enum`?

Reply:
Hmm I don't think we would be able to have the (key,value) mapping if it was an Enum.